src/main/java/com/dmytrohuk/weborganizer/calendars/Calendars.java

```diff
@@ -0,0 +1,63 @@
+package com.dmytrohuk.weborganizer.calendars;
+
+import com.dmytrohuk.weborganizer.reminders.Reminder;
+import com.dmytrohuk.weborganizer.users.User;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+@Table(name = "calendars")
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class Calendars {
```

Comment:
Entity should be a singular noun unless it is a collection of calendar

Reply:
fixed